UndertaleModTool/Windows/FindReferencesTypesDialog/UndertaleResourceReferenceMethodsMap.cs

```diff
@@ -1498,39 +1498,42 @@ public static async Task<Dictionary<string, List<object>>> GetUnreferencedObject
                 mainWindow.SetProgressBar(null, "Assets", 0, assets.Count);
                 mainWindow.StartProgressBarUpdater();
 
-                var assetsPart = Partitioner.Create(0, assets.Count);
-
                 List<Dictionary<string, List<object>>> dicts = new();
 
-                await Task.Run(() =>
+                if (assets.Count  != 0) // A Partitioner can't be created on an empty list.
```

Comment:
please fix the whitespace here.
I also personally prefer `> 0` but thats semantics.

Reply:
Done. I agree that `> 0` looks better.